### PR TITLE
Use "Artsy for Galleries" instead of "Partner with Artsy" naming in mobile nav

### PR DESCRIPTION
**Pull request by sweir27**
Addresses: https://artsyproduct.atlassian.net/browse/FX-1880

image

### CODE REVIEW

File: src/Components/NavBar/menuData.ts [hunk 1]
```diff
@@ -440,7 +440,7 @@ export const menuData: MenuData = {
       href: "/institutions",
     },
     {
-      text: "Partner with Artsy",
+      text: "Artsy for Galleries",
```
Comment (sweir27) [inline src/Components/NavBar/menuData.ts:443]: @xtina-starr Given we hard-code this in the file above, are we using this data anywhere? Wondering if we should remove it from this file or update the logic above-- what do you think?

Comment (xtina-starr) [reply src/Components/NavBar/menuData.ts:443]: Yeah I think we can remove it from this file.

**Comment by sweir27** [comment]
Alright @xtina-starr updated the screenshot! I think this is good to go.

File: src/Components/NavBar/menuData.ts (hunk 1 above)
Comment (sweir27) [reply src/Components/NavBar/menuData.ts:443]: It looks like there are a few entries that are unused. Would you mind giving it a once-over and removing the ones that aren't necessary as you're working on the desktop-side?

Comment (xtina-starr) [reply src/Components/NavBar/menuData.ts:443]: Yeah! Definitely.

**Comment by artsyit** [comment]
:rocket: PR was released in `v25.42.4` :rocket:

**Pull request merged**